mnefun/_mnefun.py

```diff
@@ -2495,6 +2495,9 @@ def plot_chpi_snr_raw(raw, win_length, n_harmonics=None):
     # SNR plots for gradiometers and magnetometers
     ax = axs[0]
     lines1 = ax.plot(tvec, 10*np.log10(snr_avg_grad.transpose()))
+    lines1_med = ax.plot(tvec, 10*np.log10(np.median(snr_avg_grad,
+                                               axis=0).transpose()),
```

Comment:
Do you still need the transpose? It is presumably 1D following the median operation. Also, you can use `.T` instead of `.transpose()`